lib/ingram_micro.rb

```diff
@@ -15,4 +16,8 @@ def self.configure
     yield configuration
     configuration.assert_valid
   end
+
+  def self.generate_order_number(prefix='')
```

Comment:
isn't the order number supposed to be an integer? i'm wondering if allowing people to pass in a prefix would make people inclined to put in a string